src/nexmo/__init__.py

```diff
@@ -121,129 +123,102 @@ def __init__(
         self.application_v2 = ApplicationV2(api_server)
 
         self.session = requests.Session()
+    
+    # Get and Set __host attribute
```

Comment:
Just a question, do we want to provide also some validation of the custom host, i.e. that it is a URL format and not a random string, and if it is a random string raise an exception for the user?

Reply:
That's actually an excellent idea! will do that!